4-Deployment/1-deploy-storage/SPA/src/App.jsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React, { useState } from "react";
+
+import { MsalProvider, AuthenticatedTemplate, UnauthenticatedTemplate, useMsal } from "@azure/msal-react";
```

Comment:
import useAccount is required, as after sign-in it throws useAccount undefined exception.